packages/app/src/providers/h5grove/__snapshots__/h5grove-api.test.ts.snap

```diff
@@ -1894,6 +1894,37 @@ exports[`test file matches snapshot 1`] = `
       1,
     ],
   },
+  {
+    "name": "vlen_float32_1D",
```

Comment:
Additional vlen dataset to test two things:

- what happens when an item in the vlen dataset is not initialized => providers should return an empty array or typed array;
- that h5wasm returns typed arrays (can't test this with the `vlen_int64_1D` dataset below since bigint typed arrays are converted to plain number arrays).

See also Python code in `create_h5_sample.py`.